core/src/jsMain/kotlin/Database.kt

```diff
@@ -46,12 +42,23 @@ public suspend fun openDatabase(
     return database
 }
 
+public suspend fun deleteDatabase(name: String) {
+    val factory = checkNotNull(window.indexedDB) { "Your browser doesn't support IndexedDB." }
+    val request = factory.deleteDatabase(name)
+    request.onNextEvent("success", "error") { event ->
+        when (event.type) {
+            "error" -> throw ErrorEventException(event)
+            else -> null
+        }
+    }
+}
+
 public class Database internal constructor(internal val database: IDBDatabase) {
 
     /**
-     * Inside the [action] block, it is important to write code which does not suspend EXCEPT for the following safe calls:
-     * - [Transaction.get]
-     * - [Transaction.getAll]
+     * Inside the [action] block, you must not call any `suspend` functions except for those provided by this library
+     * and scoped on [Transaction] (and its subclasses), and flow operations on the flows returns by [Transaction.openCursor]
+     * and [Transaction.openKeyCursor].
```

Comment:
nit: I really liked the allowable calls formatted as a list.

```suggestion
     * Inside the [action] block, you must not call any `suspend` functions except for:
     * - those provided by this library and scoped on [Transaction] (and its subclasses)
     * - flow operations on the flows returns by [Transaction.openCursor] and [Transaction.openKeyCursor]
```